lib/services/API/country_educational_institute_api.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:mobile_app/constants.dart';
+import 'package:mobile_app/models/failure_model.dart';
+import 'package:mobile_app/utils/api_utils.dart';
+
+abstract class CountryInstituteAPI {
```

Comment:
class name and file name differs, can you change file name to `country_institute_api.dart` ?